src/PKSim.R/RRegister.cs

```diff
@@ -17,6 +26,24 @@ public override void RegisterInContainer(IContainer container)
 
             scan.WithConvention<PKSimRegistrationConvention>();
          });
+         registerCLITypes(container);
+         
+      }
+
+      private static void registerCLITypes(IContainer container)
+      {
+         // container.Register<IProgressUpdater, NoneProgressUpdater>();
+         // container.Register<IDialogCreator, CLIDialogCreator>();
+         // container.Register<IDisplayUnitRetriever, CLIDisplayUnitRetriever>();
+         container.Register<IJournalDiagramManagerFactory, CLIJournalDiagramManagerFactory>();
+         container.Register<IDiagramModel, CLIDiagramModel>();
+         container.Register<IDataImporter, CLIDataImporter>();
+         container.Register<IEntityValidationTask, CLIEntityValidationTask>();
+         // container.Register<IOntogenyTask, CLIIndividualOntogenyTask>();
+         container.Register<IDiagramModelToXmlMapper, CLIDiagramModelToXmlMapper>(LifeStyle.Singleton);
+         // container.Register<IHistoryManager, HistoryManager<IExecutionContext>>();
+         // container.Register<ICoreUserSettings, OSPSuite.Core.ICoreUserSettings, CLIUserSettings>(LifeStyle.Singleton);
+         // container.Register<ICoreWorkspace, IWorkspace, CLIWorkspace>(LifeStyle.Singleton);
```

Comment:
That's just copy/paste from the CLI Register. We can refactor/combine as implementation progresses.